src/util/nondet.cpp

```diff
@@ -93,15 +82,16 @@ code_blockt generate_nondet_switch(
 
   code_blockt result_block;
 
+  allocate_objectst allocate_objects(
+    mode, source_location, name_prefix, symbol_table);
```

Comment:
:pick: prefer braces

Reply:
Done. I keep forgetting about this new convention...

Reply:
It's not really new, just that it wasn't really applied, then someone pointed out this guideline to me on one of my PR, so I have been annoying everyone with this since then. :smiling_imp: